.vscode/settings.json

```diff
@@ -2,9 +2,12 @@
   "python.formatting.provider": "none",
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
-      "source.organizeImports": true,
+    "source.organizeImports": true
   },
   "[python]": {
     "editor.defaultFormatter": "ms-python.black-formatter"
   },
+  "python.testing.pytestArgs": ["tests"],
+  "python.testing.unittestEnabled": false,
+  "python.testing.pytestEnabled": true
```

Comment:
curious: why do we need these changes? are these intentional?

Reply:
Yes, these changes are intentional. They’re for configuring the VSCode testing settings so devs who use it can immediately use the IDE's testing features with pytest, specifying the tests location and ensuring that unittest is disabled since we are not using that testing framework

Reply:
sounds good, makes sense to me.